packages/devtools_app/lib/src/screens/debugger/call_stack.dart

```diff
@@ -112,17 +114,15 @@ class _CallStackState extends State<CallStack>
       ),
     );
 
-    if (isAsyncBreak) {
-      return result;
-    } else {
-      return DevToolsTooltip(
-        message: locationDescription == null
-            ? frameDescription
-            : '$frameDescription $locationDescription',
-        waitDuration: tooltipWaitLong,
-        child: result,
-      );
-    }
+    return isAsyncBreak
+        ? result
```

Comment:
@kenzieschmoll what do you think of this case is. Brian thought it is a bit harder to read but I could go either way on this one.

Reply:
I also could go either way. I think I prefer the if / else here but the ternary isn't terrible since of of the options is a one-liner.